docs/tutorial/tutorial.md

```diff
@@ -34,7 +34,7 @@ We'll be using an online editor called CodePen in this guide. You can begin by o
 
 #### Following Along Locally
 
-You can also follow along locally if you don't mind a few extra steps:
+Alternatively, you can set up a project on your computer. This is more work, but gives you a standard development environment, including support for <a href="https://medium.freecodecamp.com/javascript-modules-a-beginner-s-guide-783f7d7a5fcc" target="_blank">modules</a>.
```

Comment:
I put a lot of thought into this piece, but I don't think it's quite right. My thoughts:

I think "alternatively" is a clear enough indicator that this isn't required.

"standard development environment" is presumptuous, but I think it's important to emphasize that this reflects the way React apps are developed.

Understanding modules is an important part of normal React app development, but maybe we shouldn't send readers off in that direction at this point.